symengine/tests/cwrapper/test_cwrapper.c

```diff
@@ -452,6 +452,17 @@ void test_constants() {
 
 }
 
+void test_ascii_art() {
+    char* s = ascii_art_str();
+    char expect[] = " _____           _____         _         \n"
+                    "|   __|_ _ _____|   __|___ ___|_|___ ___ \n"
+                    "|__   | | |     |   __|   | . | |   | -_|\n"
+                    "|_____|_  |_|_|_|_____|_|_|_  |_|_|_|___|\n"
+                    "      |___|               |___|          \n";
+    SYMENGINE_C_ASSERT(strcmp(s, expect) == 0);
```

Comment:
I would not test the actual contents of the string, just test that the returned string has a non-zero length. I think it's redundant to test the contents.

E.g. here I posted some better designs that I personally liked more:
https://github.com/symengine/symengine/pull/414#discussion_r32447845
So that it's easy for us to change it at one place.